alws/crud/build_node.py

```diff
@@ -298,8 +300,41 @@ def append_errata_package(_, errata_package, artifact, rpm_info):
             )
         rpms.append(artifact)
 
+    def is_source_rpm_included(
+            rpm_pkg: dict,
+            pgks_info: dict,
+            current_module: ModuleWrapper
+    ) -> bool:
+        source_rpm_list = [
```

Comment:
This will be compiled 2N times where N is a number of packages for processing. Maybe compile this once and pass into this function?